lib/chef/dsl/powershell.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Author:: Jay Mundrawala (<jdm@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'chef/util/powershell/ps_credential'
+
+class Chef
+  module DSL
+    module Powershell
+      def ps_credential(username='placeholder', password)
```

Comment:
@jdmundrawala  @btm The in-box credentials aren't serializing the full credential in DSC.  They just use the credential as a container to secure the password and a regular string for the password.  I don't know why - it must have made sense to them at some point, though that is not the case in every usage.  Community or other resources could expect a valid credential to come back with a username.